### ISSUE TITLE
Mod not showing in game

### ISSUE THREAD

**Issue by CaptPungent** [embarq/CSII-Clock-widget-mod#1]
Hi. Thank you for your hard work. I installed this mod, however, it doesn't show in game and isn't listed in options.

**Comment by embarq** [comment]
Hi. that's right - the mod doesn't have options that's why it doesn't show up. The only place it's currently listed is in your paradox mod playset

**Comment by Joymingming** [comment]
doesn't show in game

**Comment by BoldArtistix** [comment]
Indeed, the mod installs and activates in the playset correctly. However, the clock does not display in the bottom-right (or anywhere else).
I'm playing at medium resolution, 1920x1200, borderless fullscreen.
